.github/workflows/ci.yml

```
@@ -0,0 +1,116 @@
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        rust: [stable, beta]
+        exclude:
+          - os: windows-latest
+            rust: beta
+          - os: macos-latest
+            rust: beta
+
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)